exporter/trace/stackdriver/stackdriver.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2019, OpenTelemetry Authors
+// Copyright 2017, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stackdriver
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"time"
+
+	traceapi "cloud.google.com/go/trace/apiv2"
+	"go.opentelemetry.io/api/key"
+	"go.opentelemetry.io/sdk/trace"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/option"
+)
+
+// Options contains options for configuring the exporter.
+type Options struct {
```

Comment:
With the Jaeger exporter moving to functional options (#161), how about doing the same here?

Reply:
Thank you for the suggestion. I changed the code to use functions instead of passing struct directly to the NewExporter function.